packages/xarc-app-dev/src/lib/app-dev-middleware.ts

```diff
@@ -99,7 +99,7 @@ export class AppDevMiddleware {
   }
 
   setup({ serverPort = undefined } = {}) {
-    process.on("message", data => {
+    process.on("message", (data: any) => {
```

Comment:
lets define this a bit more?
```ts
type ProcessDataIncoming = {
  name: string;
  [x: string | symbol | number]: unknown;
}
```

```ts
    process.on("message", (data: ProcessDataIncoming) => {
```
or inline...
```ts
    process.on("message", (data: { name: string; [x: string | symbol | number]: unknown; }) => {
```

Reply:
Sounds good. Let me create a PR 👍